app/views/posts/show.html.erb

```diff
@@ -1 +1,8 @@
-<h1>Specific post for a specific user</h1>
\ No newline at end of file
+<div class="all-comments-container">
+    <%= render partial: 'posts/post', locals: { post: @post } %>
+    <div class="list-of-comments">
+        <% @post.comments.each do |comment| %>
+            <p class="post-comment"><%= @user.name %>: <%= comment.text %></p>
+        <%end%>
+    </div>
+</div>
```

Comment:
- [ ] Kindly improve the post part of the page to fit the wireframe.

  Your Implementation | Wireframe
  :---:|:---:
  ![image](https://github.com/nilab-khwaja/blog-app/assets/54848363/08be2c02-ce58-488d-b0ca-a3c8e6bc5d17) | ![image](https://github.com/nilab-khwaja/blog-app/assets/54848363/4ac1f7c0-e6e1-423d-9a63-d644cabd416a)